docs/design/latency_definitions/communication.md

```diff
@@ -0,0 +1,170 @@
+# Communication
+
+The communication latency is the time difference between the time the message is passed from the ownership of the callback to the next callback.
+
+$$
+l_{comm} = t_{receive} - t_{transfer}
+$$
+
+In particular, if transfer of ownership is publish and receive of ownership is subscribe, it is defined as follows
+
+$$
+l_{comm} = t_{sub} - t_{pub}
+$$
+
+<prettier-ignore-start>
+!!! Info
+        In this definition, communication latency is now affected by scheduling.
+        It is not only the communication latency of the DDS.
+        If multiple callbacks can be executed at the same time, it may include the execution time of other callbacks.
```

Comment:
I cannot understand the intention of this sentence.

Reply:
removed 
28579f5